### PR TITLE
Added button to Admin Dashboard to Navbar

### PR DESCRIPTION
**Pull request by julianrkung**
Added button on Navbar that directs to AdminDashboard. Also cleaned up a couple lines of extra code in Navbar.

image

CC: @didvi

### CODE REVIEW

**Comment by didvi** [comment]
Can move the button so the sign out button is first, then the admin board, then the home icon

**Comment by julianrkung** [comment]
Done ^. Also fixed the styling bug with the uncentered text on small window sizes. I'll change the navbar so that it is responsive up until 60px in the [navbar style PR](https://github.com/calblueprint/unloop/pull/65) rather than this one.

File: app/views/staffs/dashboard.html.erb [hunk 1]
```diff
@@ -1,2 +1,2 @@
 <%= link_to 'Admin', rails_admin_path if @user.admin? %>
```
Comment (wilsonw926) [inline app/views/staffs/dashboard.html.erb]: Can we remove this now?

Comment (wilsonw926) [inline app/views/staffs/dashboard.html.erb]: Can we remove this now?

Comment (julianrkung) [reply app/views/staffs/dashboard.html.erb]: yee done ty

**Pull request merged**